dev/ci/ci-user-overlay.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Add user overlays here. You can use some logic to detect if you are
+# in your travis branch and conditionally enable the overlay.
```

Comment:
That being said, I am not sure this approach is a good idea at all, but I couldn't imagine a better one for the moment...